CMakeLists.txt

```diff
@@ -35,6 +39,7 @@ include_directories(
   include
```

Comment:
I'm going to point you to one of my other packages for ros2 cmakelist best practices, this is really not easily maintainable with all the include / link dirs. If you set a variable with all the dependencies and you use ament, you can get rid of most of this code.

https://github.com/ros-planning/navigation2/blob/master/nav2_planner/CMakeLists.txt